articles/email/index.md

```diff
@@ -5,172 +5,27 @@ url: /email
-
-You can modify the lifetime of this link for security purposes. By default, the lifetime is 432000 seconds (which is equivalent to five days).
-
-If users click on an expired link and a **Redirect To** URL is configured, they will be redirected to the configured **Redirect To** URL. The following text will be appended to the querystring:
-
-```text
-http://myapplication.com/my_page/?email=john%contoso.com&message=Access%20expired&success=false
-```
-
-## Email Templates
-
-### Verification Email
-
-When users sign up or log in for the first time, they will be sent a verification email. Clicking on the verification link in the email sets the 'email verified' property of their user profile to `true`.
-
-The following macros are available in the **Verification Email** template:
-
-* `{application.name}`
-* `{connection.name}`
-* `{user.email}`
-
-If you configure a **Redirect To** URL, the user will be directed to this URL after clicking the verification link. The following will be appended to the querystring:
-
-```text
-http://myapplication.com/my_page/?email=john%40contoso.com&message=Your%20email%20was%20verified.%20You%20can%20continue%20using%20the%20application.&success=true
-```
-
-### Welcome Email
-
-Once a user verifies their email address, they will receive a **Welcome Email**. If you turn off the **Verification Email** feature, the **Welcome Email** will be sent to the user when they sign up (or login for the first time).
-
-The following macros are available in the **Welcome Email** template:
-
-* `{application.name}`
-* `{connection.name}`
-* `{user.email}`
-
-### Change Password Confirmation Email
-
-If a user requests a password change, they will receive a **Change Password Confirmation Email**. Until the user clicks the verification link contained in the email, the password will remain unchanged.
-
-The following macros are available in the **Change Password Confirmation** email template:
-
-* `{application.name}`
-* `{connection.name}`
-* `{user.email}`
-
-If you configure a **Redirect To** URL, the user will be directed to this URL after clicking the verification link. The following will be appended to the querystring sent:
-
-```text
-http://myapplication.com/my_page/?success=true&message=You%20can%20now%20login%20to%20the%20application%20with%20the%20new%20password.
-```
-
-### Blocked Account Email
+[Click here to learn more about Email Delivery Changes](/migrations#email-delivery-changes)
+:::
 
-If a user attempts and fails to log in ten or more times from a specific IP address, the user's account becomes locked and they will receive a **Blocked Account** email. Once the user receives this email, they will not be able to log in from that IP address again until they click on the link contained in the email.
+Auth0 provides built-in email services to easily communicate with your users. This includes, verification emails, welcome emails, change password emails and blocked account emails.
```

Comment:
Also I prefer the trailing comma: `change password emails, and blocked account emails.`